website/docs/reference/resource-properties/freshness.md

```diff
@@ -87,6 +87,8 @@ This is particularly useful if:
 - You are using BigQuery and your source tables are [partitioned tables](https://cloud.google.com/bigquery/docs/partitioned-tables)
 - You are using Snowflake or Spark with large tables, and this results in a performance benefit
 
+TODO: should this be Databricks instead of Spark?
```

Comment:
If we are to change this - perhaps we just include Databricks in the call out since it's not untrue for Spark?

Reply:
```suggestion

```